backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -123,12 +67,14 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
   /**
    * Create a MimeMessage using the parameters provided.
    *
-   * @param subject subject of the email
+   * @param recipientEmail email of user to whom is to be notified
```

Comment:
email to which the notification should be sent ?

Reply:
Ack